framework/commercetools/lib/normalize.ts

```diff
@@ -0,0 +1,109 @@
+import type {
+  CommercetoolsProducts,
+  CommerceToolsProductPrice,
+  Product,
+  ProductVariant,
+  CommercetoolsProductVariant,
+  ProductPrice,
+  Images,
+  ProductImage,
+} from '../types/product'
+import type {
+  Cart,
+  CommercetoolsCart,
+  CommercetoolsLineItems,
+  LineItem,
+} from '../types/cart'
+
+function setProductPrice(prices: CommerceToolsProductPrice): ProductPrice {
+  return {
+    value: prices.value.centAmount,
+    currencyCode: prices.value.currencyCode,
+    retailPrice: 0,
+    salePrice: 0,
```

Comment:
Maybe this value can be set depending if the price has a discounted value. Inside a price you can check the `discounted` field.